GrandCentralBoard.xcodeproj/project.pbxproj

```diff
@@ -1247,6 +1250,7 @@
 				1A87495E1C385264006E58C6 /* RemoteImageSource.swift in Sources */,
 				F9C88F3E1CBFC72100E81A14 /* AccessTokenFetcher.swift in Sources */,
 				52FB50621CBE862D0053DE8B /* EventsSource.swift in Sources */,
+				F928B82E1CC52F77006D0332 /* String+Localized.swift in Sources */,
```

Comment:
It seems that most extension files use the Objective-C category naming pattern, i.e. Class+Category.extension